test/blame/b-key-navigation-test

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+. libtest.sh
+. libgit.sh
+
+export COLUMNS=200
+
+# This test verifies the fix for issue #1369
+# The test checks that pressing 'b' in blame view correctly navigates
+# to the corresponding line in the target commit.
+
+in_work_dir create_repo_from_tgz "$base_dir/files/scala-js-benchmarks.tgz"
+
+tigrc <<EOF
+set line-graphics = ascii
+EOF
+
+steps '
+	# Press b to go to blame for that line/commit
+	:view-blame
+	:save-display recursive-blame.screen
+'
+
+test_tig blame +27 project/Build.scala
+
+# This shows that after pressing 'b' we should be positioned the line starting
```

Comment:
Also not really a problem, but this confused me at first. It'd help if you at least added the expected line number and the fact that it's shown in the status bar.

Also, there's a typo, "on" is missing after "positioned".

Here's the comment I've come up with for the corresponding spot in #1372:

> This shows that after pressing ',' we should be positioned on line 40 (starting with "clean in common"), as shown in the status bar. We should *not* stay on line 36.